pkg/cluster/internal/kubeadm/config.go

```diff
@@ -427,6 +427,147 @@ conntrack:
 {{end}}{{end}}
 `
 
+// ConfigTemplateBetaV2 is the kubeadm config template for API version v1beta2
+const ConfigTemplateBetaV3 = `# config generated by kind
+apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
i guess, apiVersion and code comments should be for V3 / v1beta3.

Reply:
LOL sigh, fixed 🙃